apps/frontend/src/main.tsx

```diff
@@ -63,22 +64,43 @@ const router = createBrowserRouter([
                     },
                     {
                         path: "projects/:projectSlug/surveys",
-                        element: <Surveys />,
+                        element: (
```

Comment:
@kaf-lamed-beyt This is incorrect. All survey related components should be under a single `SurveyProvider`. Did you try the code I shared with you earlier?

Reply:
Yes, I did. 

SurveyProvider expects a `children` prop. Appending it like so: `<SurveyProvider />` throws an error. Can't remember the exact error in the console. But, the page became blank, nothing rendered.

Reply:
@kaf-lamed-beyt Because you need an `Outlet` from `react-router-dom`. That should be its child. So it should be like;

```export default function SurveyContainer() {
    return (
        <SurveyProvider>
                <Outlet />
        </SurveyProvider>
    );
}```

Then use `SurveyContainer` in `main.tsx`.

Reply:
Aha! I see. Thank you.